firmware/arduino/chessboard/circleFunction.ino

```diff
@@ -0,0 +1,250 @@
+int pulsesPerSlope[NUM_CIRCLES][NUM_SLOPES_PER_QUARTER_CIRCLE * 2 - 2];
+
+enum SlopeToIndex
+{
+    SLOPE_HORIZONTAL_MAX_SIZE = 0,
+    SLOPE_HORIZONTAL = 1,
+    SLOPE_EIGHTH = 2,
+    SLOPE_QUARTER_MAX_SIZE = 3,
+    SLOPE_QUARTER = 4,
+    SLOPE_HALF_MAX_SIZE = 5,
+    SLOPE_HALF = 6,
+    SLOPE_ONE_MAX_SIZE = 7,
+    SLOPE_ONE = 8,
+    SLOPE_TWO_MAX_SIZE = 9,
+    SLOPE_TWO = 10,
+    SLOPE_FOUR_MAX_SIZE = 11,
+    SLOPE_FOUR = 12,
+    SLOPE_EIGHT = 13,
+    SLOPE_VERTICAL_MAX_SIZE = 14,
+    SLOPE_VERTICAL = 15
+};
```

Comment:
Can you add a comment to the `SlopeToIndex` enum to explain that some of these are paired together as a combo of fast/fine-tune travel, e.g. SLOPE_HORIZONTAL_MAX_SIZE and SLOPE_HORIZONTAL are used together. Some way to clarify why you have two SLOPE_HORIZONTAL type indices. Also maybe you can change name from above to SLOPE_HORIZONTAL_FAST and SLOPE_HORIZONTAL_SLOW and similar elsewhere. Also, you can remove the SLOPE_ prefix since they all share it and the rest of the code is still readable without the prefix

Reply:
I like the fast/slow naming convention, but I'd prefer to keep the SLOPE_ prefix. I think it would be odd for ONE to be 7.